.github/workflows/lib-injection.yml

```diff
@@ -48,63 +48,117 @@ jobs:
       id: buildx
       uses: docker/setup-buildx-action@v2
 
-    - name: Generate libc image tags
+    - name: lib-injection-tags
```

Comment:
Dumb question but we don't use the output anymore, so do we need to run it?

Reply:
You're absolutely right, I'll get rid of it

Reply:
Fixed in 3887ee97cb3dec1b25ab2636f78605526424efcb